packages/wmr/test/production.test.js

```diff
@@ -878,6 +878,17 @@ describe('production', () => {
 				expect(instance.output.join('\n')).toMatch(/The following error was thrown during prerendering/i);
 			});
 		});
+
+		it('should support prerendered HTML with markdown using globalThis.fetch', async () => {
+			await loadFixture('prerender-markdown-content', env);
```

Comment:
nit: The focus of this test is less the markdown and instead the resource fetching. Markdown's a great & common placeholder, but not overall important to the test. What do you think of this?

```suggestion
		it('should support fetching resources from disk during prerender', async () => {
			await loadFixture('prerender-resource-fetch', env);
```

Directory name will need to be updated too if you agree. Sadly I can't make a suggestion with file paths.

Reply:
I felt that was indeed the case.
I named it markdown because I happen to use markdown files for the resources I use for testing, and I agree with you about the naming.
I will make the correction.